blockly-core/core/ui/css.js

```diff
@@ -132,7 +98,10 @@ Blockly.Css.setCursor = function(cursor, opt_svg) {
  */
 Blockly.Css.CONTENT = [
   '.blocklyDraggable {',
-    // Placeholder for cursor rule. Must be first rule (index 0).
+  '  cursor: url(/blockly/media/handopen.cur) 8 5, auto;',
+  '}',
+  '.dragging, .dragging .blocklySvg, .dragging .blocklyDraggable {',
+  '  cursor: url(/blockly/media/handclosed.cur) 7 3, auto !important;',
```

Comment:
Should these use `Blockly.assetUrl('media/*.png')` ? Noticing on this branch locally the cursors don't work on the blockly-core/tests/playground.html page